qa/util/src/main/java/io/camunda/qa/util/multidb/CamundaMultiDBExtension.java

```diff
@@ -269,14 +271,17 @@ private List<User> findUsers(
     return users;
   }
 
-  private void injectFields(
-      final Class<?> testClass, final Object testInstance, Predicate<Field> predicate) {
-    predicate = predicate.and(field -> field.getType() == CamundaClient.class);
+  private void injectStaticClientField(final Class<?> testClass) {
     for (final Field field : testClass.getDeclaredFields()) {
       try {
-        if (predicate.test(field)) {
-          field.setAccessible(true);
-          field.set(testInstance, createCamundaClient(field.getAnnotation(Authenticated.class)));
+        if (field.getType() == CamundaClient.class) {
+          if (ModifierSupport.isStatic(field)) {
+            field.setAccessible(true);
+            field.set(null, createCamundaClient(field.getAnnotation(Authenticated.class)));
+          } else {
+            fail(
+                "Client field didn't match predicate, and couldn't be injected. Make sure it is static.");
```

Comment:
🙃 Maybe we can make this more tangible. But the current message is fine as well.
```suggestion
                "Camunda Client field couldn't be injected. Make sure it is static.");
```